src/Core/Content/Test/ProductExport/Service/ProductExporterTest.php

```diff
@@ -58,36 +56,12 @@ public function testExport(): void
         $this->service->export($this->salesChannelContext, new ExportBehavior());
 
         $filePath = sprintf('%s/Testexport.csv', $this->getContainer()->getParameter('product_export.directory'));
-        $fileContent = $this->fileSystem->read($filePath);
-
-        static::assertIsString($fileContent);
-        $csvRows = explode(\PHP_EOL, $fileContent);
 
         static::assertTrue($this->fileSystem->directoryExists($this->getContainer()->getParameter('product_export.directory')));
         static::assertTrue($this->fileSystem->fileExists($filePath));
-        static::assertCount(4, $csvRows);
-    }
-
-    public function testExportPagination(): void
```

Comment:
why did you remove this test?

Reply:
As it is actually the same as the `testExport()` when the "useles" assert is removed :-)